README.md

```diff
@@ -514,7 +514,7 @@ Thus when you create a TODO, it is almost always your name that is given.
      end
      ```
 
-* <a name="no-default-args"></a>Do not use default arguments. Use an options
+* <a name="no-default-args"></a>Do not use default positional arguments. Use named arguments or an options
```

Comment:
@kmsun07 so you mean default positional args must appear at the end of the function signature, but default keyword args can appear anywhere? Like this:
``` def obliterate(gently:, at: Time.now, another_arg:) ```
but we can't do this:
``` def obliterate(gently, at=Time.now, another_arg) ``` ?

I think this only applies to Ruby 2.1 and beyond:
https://robots.thoughtbot.com/ruby-2-keyword-arguments

Why is it bad though? Isn't the main reason that it creates obscurity from an ordering standpoint?